app.rb

```diff
@@ -146,9 +146,19 @@ def authorization_url(ial:, aal: nil)
         state: random_value,
         nonce: random_value,
         prompt: 'select_account',
-      }.to_query
+      }
 
-      "#{endpoint}?#{request_params}"
+      request_params.merge!(user_options)
+
+      "#{endpoint}?#{request_params.to_query}"
+    end
+
+    def user_options
+      ip_auth_option = params['ip_auth_option']
+
+      {}.tap do |options|
+        options[:inherited_proofing_auth] = random_value if ip_auth_option
+      end
```

Comment:
Do we need the option to be a random value? I can see how other uses of `random_value` should in-fact be random, but in this case it seems like we could just pass through the form value directly, if we're mostly concerned with checking its presence on the IdP end?

```suggestion
      params.slice('ip_auth_option')
```

Reply:
I think the form value is: true. My co-developer pointed out that, we actually have a VA test value (mocked-auth-code-for-testing), so I'm inclined to use that, instead.